### ISSUE TITLE
Maven Central artifact publishing

### ISSUE THREAD

**Issue by Vladislav-Zolotaryov** [Nothing-Developer-Programme/Glyph-Developer-Kit#1]
Hello!

Really cool that you provided a Dev Kit for the Glyph.
Can you please also publish it to [maven central repository](https://mvnrepository.com/repos/central) for ease of project setup?

Thank you very much!

**Comment by CrazyMarvin** [comment]
Yes, an artifact is needed so that [Flashy ](https://github.com/Crazy-Marvin/Flashy/issues/205)may implement this.
